### PR TITLE
Add quickstart

### PR DESCRIPTION
**Pull request by WyriHaximus**
https://github.com/WyriHaximus/TwigView/issues/20#issuecomment-231276824

### CODE REVIEW

**Comment by htstudios** [comment]
You rock!
Btw did you check out Elixir (I was that dude at Cake Fest)

File: quickstart.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+## Change AppView
+```PHP
+// src/View/AppView.php
+namespace App\View;
+
+use Cake\View\View;
```
Comment (htstudios) [inline quickstart.md]: Is that required?

File: quickstart.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+}
+```
+
+## Load Helpers
+```PHP
+// src/Controller/AppController
+
+class AppController extends Controller
+{
+    public $helpers = ['Html', 'Form']; // and mode
```
Comment (htstudios) [inline quickstart.md]: This  could possibly be loaded in AppView::initialize() or via viewBuilder maybe?

File: quickstart.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+    <section>
+        <h1 class="page-title">{{ _view.fetch('title') }}</h1>
+        {{ _view.fetch('content')|raw }}
+    </section>
+
+</body>
+```
+
+## Create a view
```
Comment (htstudios) [inline quickstart.md]: Create a view template

File: quickstart.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+// src/Controller/AppController
+
+class AppController extends Controller
+{
+    public $helpers = ['Html', 'Form']; // and mode
+
+   // code ....
+```
+
+## Replace layout default.ctp by default.tpl
```
Comment (htstudios) [inline quickstart.md]: '## Create the default layout to be used by TwigView'

Comment (htstudios) [inline quickstart.md]: ... named 'default.tpl' instead of default.ctp

**Comment by WyriHaximus** [comment]
Thanks! Just fixed the issues you pointed out :+1: 
(Not yet, been a real busy month between CakeFest and DPC. Planning to play with it this weekend :+1: .)

**Pull request merged**